### PR TITLE
Adding first street foundation data

### PR DESCRIPTION
**Pull request by emma-nechamkin**

### CODE REVIEW

**Comment by emma-nechamkin** [comment]
Right now, this is only adding the data sources.  They are NOT integrated into scores in any way because I want to minimize merge conflicts.

File: data/data-pipeline/data_pipeline/etl/score/config/datasets.yml [hunk 1]
```diff
@@ -129,3 +129,84 @@ datasets:
         include_in_tiles: true
         include_in_downloadable_files: true
 
+  - long_name: "First Street Foundation Flood Risk"
```
Comment (esfoobar-usds) [inline data/data-pipeline/data_pipeline/etl/score/config/datasets.yml:132]: LOVE you're doing YAML like a baws.

File: data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+# pylint: disable=unsubscriptable-object
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/sources/fsf_flood_risk/etl.py:1]: Note (namely to @lucasmbrown-usds) -- these two datasets get munged basically identically. We could move the work into a separate util function for transform, but the yaml file means that we wouldn't really save any lines of code. LMK if you think this explicit work is the wrong call.

**Review by esfoobar-usds — APPROVED** [review]
YAML 🚀 for Nechamkin

**Comment by emma-nechamkin** [comment]
The test failures we have here are ones Matt is working through.

**Pull request merged**